Autopilot/PathManager/Inc/cruisingState.hpp

```diff
@@ -0,0 +1,40 @@
+/*
+* Declares the functions that interact with the Waypoint Manager (on behalf of the cruisingState state)
+* Author: Dhruv Rawat
+*/
+
+#include "pathDatatypes.hpp"
+#include "waypointManager.hpp"
+
+#ifndef CRUISING_STATE
+#define CRUISING_STATE
+
+enum _ModifyFlightPathErrorCode { MODIFY_CRUISING_SUCCESS = 0, MODIFY_CRUISING_ERROR, MODIFY_CRUISING_INCORRECT_TELEMETRY_COMMAND };
+enum _GetNextDirectionsErrorCode { PATH_CRUISING_SUCCESS = 0, PATH_CRUISING_ERROR, PATH_CRUISING_INCORRECT_TELEMETRY_COMMAND, PATH_CRUISING_UNINITIALIZED_HOMEBASE };
+
+struct _CruisingState_Telemetry_Return {
+    char editingFlightPathErrorCode; // 0 = success, 1 = error, 2 = incorrect telemetry command
+    char pathFollowingErrorCode; // 0 = success, 1 = error, 2 = home base not initialized, 3 = incorrect telemetry command
+    int currentWaypointId; 
+    int currentWaypointIndex;
+    bool homeBaseInitialized;
+};
+
+// The following functions are used to interact with the Waypoint Manager module
```

Comment:
A description of what each function does, how it needs to be called and what the arguments/return values are is needed